lib/Doctrine/Import/Builder.php

```diff
@@ -782,7 +782,7 @@ public function buildPhpDocs(array $definition)
             if (isset($definition['relations']) && ! empty($definition['relations'])) {
                 foreach ($definition['relations'] as $relation) {
                     $type = (isset($relation['type']) && $relation['type'] == Doctrine_Relation::MANY) ? 'Doctrine_Collection' : $this->_classPrefix . $relation['class'];
-                    if ($relation["type"] == Doctrine_Relation::ONE) {
+                    if (isset($relation['type']) && $relation["type"] == Doctrine_Relation::ONE) {
```

Comment:
This break the actual behaviour where when the `type` key does not exists the relation is `to-one` per default.

---

```php
0 === Doctrine_Relation::ONE
/* !isset($relation["type"]) => */ null === $relation["type"]

// Then
null == 0 // true
```

Reply:
Fixed.